src/Builder.php

```diff
@@ -220,9 +220,10 @@ public function get()
      * @param  int  $perPage
      * @param  string  $pageName
      * @param  int|null  $page
+     * @param  string  $queryName
      * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator
      */
-    public function paginate($perPage = null, $pageName = 'page', $page = null)
+    public function paginate($perPage = null, $pageName = 'page', $page = null, $queryName = 'query')
```

Comment:
On Eloquent builder `User::paginate(10)` there is a query parameter by default? Otherwise I would say to don't append nothing on the scout builder and let the user decide, e.g: `User::search($query)->paginate()->appends('q', $query);`.

Reply:
Good remark! Didn't think of that.

Reply:
why was this closed? This is an issue integrating with elastic search.. it should be possible to specify the query label

Reply:
@x1c0 Elastic search isn't supported anymore

Reply:
@driesvints I don't understand.. what that means? Laravel does not support integration with elastic search?

Reply:
Scout now only supports Algolia.